.buildkite/common.py

```diff
@@ -13,14 +13,14 @@
 import subprocess
 from pathlib import Path
 
-DEFAULT_INSTANCES = [
```

Comment:
Could use a StrEnum, but 3.11 only :/

```
class InstanceType(StrEnum):
    m5d = "m5d.metal"
    m6g = "m6g.metal"
    m7g = "m7g.metal"

    @property
    def arch(self):
        if self.value[2] == 'g':
            return "aarch64"
        return "x86_64"
```

it may be too magic anyway, nevermind, we can go there if this becomes more complex.